### ISSUE TITLE
String variables

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#337]
@manerotoni 

Will you explain [in variables](https://neubias.github.io/training-resources/variables/index.html) what a String is? 

If not, we have to explain it [in string concat](https://neubias.github.io/training-resources/string_concat/index.html).

My instinct would be to explain the most common types of variables (including strings) in the module about variables. What do you think?

**Comment by manerotoni** [comment]
I can. The variable module is still draft stage, but it make sense to explain what different variables we can have.

I feel I should also show some more advanced languages, to point that depending on the language you declare a type too. This is technical but quite important. 
  
Numbers, strings, ..

**Comment by tischi** [comment]
> I feel I should also show some more advanced languages, to point that depending on the language you declare a type too. This is technical but quite important.

Yes, good idea!

**Comment by manerotoni** [comment]
Yes maybe working with strings is more appropriate than just string concat. 
Anything like getting file names etc is working with strings.
